src/ethers/eth.js

```diff
@@ -53,6 +58,17 @@ async function getWallet(isFlashbotWallet = false) {
   return createWallet(process.env.MNEMONIC)
 }
 
+function getFlashBotGasPrice() {
+  return getProvider()
+    .getFeeData()
+    .then(function (feeData) {
+      const gasPrice = feeData.gasPrice.gt(BN.from(config.vesper.maxTxnGasPrice))
+        ? config.vesper.maxTxnGasPrice
+        : feeData.gasPrice
+      return BN.from(gasPrice).mul(BN.from(125)).div(100).toString()
```

Comment:
Ethers BN support operation on numbers and string. 
```suggestion
      const gasPrice = feeData.gasPrice.gt(config.vesper.maxTxnGasPrice)
        ? config.vesper.maxTxnGasPrice
        : feeData.gasPrice
      return BN.from(gasPrice).mul(125).div(100).toString()
```